### ISSUE TITLE
Filter: Parsing CSS   failed - aria-invalid  pseudo-class

### ISSUE THREAD

**Issue by xmlking** [keenthemes/reui#43]
trying new `filters` component in my existing shadcn project and got this startup error 

```
Parsing CSS source code failed
  5126 |   }
  5127 |   .has-\[\[data-slot\=filters-input\]\:aria-invalid\]\:border {
> 5128 |     &:has(*:is([data-slot=filters-input]:aria-invalid)) {
       |                                          ^
  5129 |       border-style: var(--tw-border-style);
  5130 |       border-width: 1px;
  5131 |     }

'aria-invalid' is not recognized as a valid pseudo-class. Did you mean '::aria-invalid' (pseudo-element) or is this a typo?

Import trace:
  Client Component Browser:
    ./apps/console/src/app/globals.css [Client Component Browser]
    ./apps/console/src/app/layout.tsx [Server Component]
```
<img width="1113" height="552" alt="Image" src="https://github.com/user-attachments/assets/693a1879-5dcf-4a03-9638-fe2c56a3f634" />

**Comment by sean0205** [comment]
Hi,

Appreciate your feedback. The issue has been fixed! 

Regards,
Sean
